modules/ROOT/pages/troubleshoot-installation-issues.adoc

```diff
@@ -0,0 +1,156 @@
+
+. Verify that you are using the correct control plane. 
+. Modify the control plane by setting the SERVICEMESH_PLATFORM_URI variable:
++
+`export SERVICEMESH_PLATFORM_URI=https://eu1.anypoint.mulesoft.com`
+. If you are not using a US Control Plane, verify that you have exported the correct variable to point to the correct one. 
+. Obtain the correct organization or environment credentials.
+. Re-run the installer with the correct pair of credentials.
+
+[[insufficient-pod-allocation]]
+== Insufficient Pod Allocation
+
+When you start the Anypoint Service Mesh installation, the installer times out.
+
+=== Causes
+
+This issue occurs if the capacity of a pod or the total number of the pods created in the cluster exceeds the limit.
+
+
+=== Solution
+
+To resolve this error, perform the following steps:
+
+. Verify the status of the pods:
++
+`kubectl get pods --all-namespaces`
++
+All the pods in all namespaces are displayed with their status and additional details.
+. If a pod displays a “Pending” status, as illustrated in the previous screenshot, obtain additional details for the pod:
++
+`kubectl describe pods <pod_name> -n service-mesh`
+//Need a text for the screenshot here.
++
+The result of the command shows that the scheduling of the node has failed.  
+. To verify that the number of pods under the capacity section is not already filled up, run the command:
++
+`kubectl describe nodes`
+. Increase capacity of the node or amount of nodes.
++
+*Tip:* If you are using Aamzon EKS, this value is calculated from the instance type. Therefore, if you increase the instance type, the amount of pods is increased.
+. Re-run the Anypoint Service Mesh installer to complete the installation.
+
+[[service-mesh-not-found]]
+== Service Mesh Not Found
+
+When you start the Anypoint Service Mesh installation, the installer fails with the “Service Mesh not found” error.
+
+=== Causes
+
+This issue can be caused due to various reasons. 
+
+=== Diagnose
+
+To identify the command that caused the error, run the following command:
++
+`$ sh -x ./service-mesh install`
+
+=== Solution
+
+To resolve this error, download the latest version of the installer and check if the problem persists.
+
+
+[[failed-webhook-calling]]
+== Installation Failed Due to Failed WebHook Calling Error
+
+When you start the Anypoint Service Mesh installation, the installer fails with the following error:
+
+[source,text,linenums]
+----
+Error: release service-mesh failed, and has been uninstalled due to atomic being set: Internal error occurred: failed calling webhook "pilot.validation.istio.io": Post https://istio-galley.service-mesh.svc:443/admitpilot?timeout=30s: no endpoints available for service "istio-galley"
+FATAL: Installing 'service-mesh' Helm Chart!!!
+----
+
+=== Diagnose
+
+This issue can be caused due to various reasons. 
+
+To diagnose the issue:
+
+* Case 1: Verify that the master nodes have access to port 443 in worker nodes. 
```

Comment:
```suggestion
* Verify that the master node has access to port 443 in worker nodes. 
```
We don't tell them how to do this?

Reply:
Yes, no steps. Fairly easy task to perform.